proxy/src/schema/registry.rs

```diff
@@ -91,6 +97,7 @@ impl Registry {
         author: &ed25519::Pair,
         domain: String,
         name: String,
+        maybe_pid: Option<String>,
```

Comment:
Interesting prefix choice, not quite sure how this is usually advocated in Rust.

Reply:
I saw the `maybe_` prefix used in a piece of example Futures code and it stuck in my patterns, but if there's a more appropriate or idiomatic expression I'd be happy to adopt it.

Reply:
Thanks for the pointer, maybe we should start gathering these little best-practices somewhere.